### DISCUSSION TITLE
Any API or way to get the score/ranks of user?

### DISCUSSION THREAD

**Discussion by Chepycou** [streetcomplete/StreetComplete#4361] [Q&A] [answered]
Hello there, 👋
I'm the president of the computer science association of my engineering school. One of our main goals is to promote open source solutions, and we are trying to get people to use/contribute to OSM. To do this, we would like to create a scoreboard for Street Complete scores. Is there an API to get the score/rank for specific users?
Thanks a lot !

**Comment by mnalis** [comment, marked as answer]
Related - see issues linked at https://github.com/streetcomplete/StreetComplete/issues/3706 - there are problems associated with pushing gamification too much (especially if one does not external validation tests to detect "cheating") .

**Reply by Chepycou** [reply]
That explains it, thank you.
